clients/drcachesim/tests/burst_syscall_inject.cpp

```diff
@@ -286,13 +321,15 @@ write_system_call_template(void *dr_context)
     write_trace_entry(
         writer, test_util::make_marker(TRACE_MARKER_TYPE_SYSCALL_TRACE_END, SYS_gettid));
 
+    instrs_in_default_trace[0] = write_default_syscall_trace(dr_context, writer);
```

Comment:
This seems a little weird to have this be an array yet all uses hardcode [0]: I think it would be clearer and simpler to have it be a single var and not an array.  If it remains an array I think the code using it has to use loops.